src/main/resources/db/migration/V01__create_users_table.sql

```diff
@@ -0,0 +1,23 @@
+CREATE TABLE "_user" (
+    id BIGSERIAL,
+    firstname VARCHAR(255),
+    lastname VARCHAR(255),
+    email VARCHAR(255),
+    password VARCHAR(255),
+    role VARCHAR(20),
+    created TIMESTAMP,
+    deleted TIMESTAMP,
+    updated TIMESTAMP,
+    PRIMARY KEY(id),
+    CONSTRAINT nn_email_not_null CHECK (email IS NOT NULL),
```

Comment:
please do not write not-null constraint separately

Reply:
Done